tsconfig.json

```diff
@@ -1,30 +1,24 @@
 {
-  "include": [
-    "public/**/*",
-    "scripts/**/*",
-  ],
-  "exclude": [
-    "public/amplitude.js",
-    "public/mobileCheck.js"
-  ],
+  "include": ["public/**/*", "scripts/**/*"],
+  "exclude": ["public/amplitude.js", "public/mobileCheck.js"],
   "compilerOptions": {
-    "target": "ES2019",                               /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "ES2019" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
     "module": "ESNext",
 
-    "rootDir": "./",                                  /* Specify the root folder within your source files. */
+    "rootDir": "./" /* Specify the root folder within your source files. */,
     "moduleResolution": "node",
-    "resolveJsonModule": true,                        /* Enable importing .json files. */
+    "resolveJsonModule": true /* Enable importing .json files. */,
 
-    "allowJs": true,                                  /* Allow JavaScript files to be a part of your program. Use the 'checkJS' option to get errors from these files. */
-    "checkJs": true,                                  /* Enable error reporting in type-checked JavaScript files. */
+    "allowJs": true /* Allow JavaScript files to be a part of your program. Use the 'checkJS' option to get errors from these files. */,
+    "checkJs": true /* Enable error reporting in type-checked JavaScript files. */,
 
-    "declaration": false,                             /* Generate .d.ts files from TypeScript and JavaScript files in your project. */
-    "outDir": "./dist",                               /* Specify an output folder for all emitted files. */
-    "forceConsistentCasingInFileNames": true,         /* Ensure that casing is correct in imports. */
-    
-    "strict": true,                                   
+    "declaration": false /* Generate .d.ts files from TypeScript and JavaScript files in your project. */,
+    "outDir": "./dist" /* Specify an output folder for all emitted files. */,
+    "forceConsistentCasingInFileNames": true /* Ensure that casing is correct in imports. */,
+
+    "strict": true,
     // TODO: enable below
-    "noImplicitAny": false,                           
-    "strictNullChecks": false,
+    "noImplicitAny": true,
+    "strictNullChecks": true
```

Comment:
Thanks for turning these on. I don't think there's anything stopping us from bundling with TS as a next step (future PR)... unless I'm forgetting something about alpine?

Reply:
A bundler would be nice. But only if it's easy.